dockertest/images_unittests.py

```diff
@@ -341,6 +341,15 @@ def test_addr_no_user(self):
         self.assertEqual(user, None)
         self.assertEqual(addr, "address:1234")
 
+    def test_empty_tag(self):
+        test = "myname:"   # note the trailing colon with no actual tag name
```

Comment:
I get the impression that the bigger picture is to keep the lifespan of 1.9 as short as possible.  It's a franken-patch-beast IMHO.

CI Runs that generally pass, are more useful than CI runs with 100% coverage on 100% platforms/versions.  I think the best long-term strategy here is to spend as little time/effort as possible fussing with 1.9.  Any bugs uncovered would have fixes stuck into 1.10+ anyway.

It's okay by me to just duct-tape workaround / TestNA's all the 1.9 specialities, and concentrate on making it right for 1.10.

Does that make sense?

Remember, disabling/enabling tests and altering test configuration for platform-specific requirements is really easy to do in CI.  I have all the test configs in jinja2 templates, so adding conditionals there is almost a no-brainer.